src/mango/depository.ts

```diff
@@ -290,4 +292,35 @@ export class MangoDepository {
       payer
     );
   }
+
+  // Return the price of 1 base native unit expressed in quote native units
+  // This is the format of the on chain program input parameter
+  async getCollateralPerpPriceNativeQuotePerNativeBase(
+    mango: Mango
+  ): Promise<I80F48> {
+    const mangoCache = await mango.getCache();
+    const pmc = this.getPerpMarketConfig(mango); // perpMarketConfig
+    const pmi = pmc.marketIndex; // perpMarketIndex
+    return mango.group.getPriceNative(pmi, mangoCache);
+  }
+
+  // The side of the taker (the user)
+  async getLimitPrice(
```

Comment:
assuming these 2 methods are now required for 3.1.0

Reply:
Yes!